src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs

```diff
@@ -141,19 +141,19 @@ internal PSObject GetPSObject()
             if (psObject == null)
             {
                 psObject = new PSObject(this.cimSession);
-                psObject.Properties.Add(new PSNoteProperty(CimSessionState.idPropName, this.sessionId));
-                psObject.Properties.Add(new PSNoteProperty(CimSessionState.namePropName, this.name));
-                psObject.Properties.Add(new PSNoteProperty(CimSessionState.instanceidPropName, this.instanceId));
-                psObject.Properties.Add(new PSNoteProperty(CimSessionState.computernamePropName, this.ComputerName));
-                psObject.Properties.Add(new PSNoteProperty(CimSessionState.protocolPropName, this.Protocol));
+                psObject.Properties.Add(new PSNoteProperty(CimSessionState.IdPropName, this.sessionId));
```

Comment:
It is not follow our code conventions https://github.com/PowerShell/PowerShell/blob/master/docs/dev-process/coding-guidelines.md#naming-conventions

It should be `s_idPropName`.

But the change is not related the PR. Please revert name changes and leave only adding `readonly`